sdk/cosmos/azure-cosmos/azure/cosmos/aio/cosmos_client.py

```diff
@@ -343,22 +375,26 @@ def query_databases(
 
     @distributed_trace_async
     async def delete_database(
-        self,
-        database,  # type: Union[str, DatabaseProxy, Dict[str, Any]]
-        **kwargs  # type: Any
-    ):
-        # type: (...) -> None
+            self,
+            database: Union[str, DatabaseProxy, Dict[str, Any]],
+            **kwargs: Any
+    ) -> None:
         """Delete the database with the given ID (name).
 
         :param database: The ID (name), dict representing the properties, or :class:`DatabaseProxy`
             instance of the database to delete.
-        :type database: str or dict(str, str) or ~azure.cosmos.DatabaseProxy
-        :keyword str session_token: Token for use with Session consistency.
-        :keyword dict[str,str] initial_headers: Initial headers to be sent as part of the request.
-        :keyword str etag: An ETag value, or the wildcard character (*). Used to check if the resource
+        :type database: str or Dict[str, str] or ~azure.cosmos.DatabaseProxy
+        :keyword session_token: Token for use with Session consistency.
+        :paramtype session_token: str
```

Comment:
```suggestion
        :keyword str session_token: Token for use with Session consistency.
```

This can simplify a lot of your docstrings.